src/homepageExperience/containers/HomepageContainer.tsx

```diff
@@ -53,6 +62,18 @@ export const HomepageContainer: FC = () => {
 
   const squareGridCardSize = '200px'
 
+  useEffect(() => {
+    dispatch(getTelegrafs)
+  }, [telegrafs])
+
+  const telegrafLink = () => {
+    if (telegrafs.length != 0) {
```

Comment:
It's generally better to use strict equality with JavaScript, so like

```ts
telegrafs.length !== 0
```
because in JavaScript `"0" == 0`

It's not a huge deal with `.length` since it'll be a number, but it's a good habit to get into as a default, so that in the cases where you actually need `"0"` to equal `0`, the comparison `myValue == 0` can be appropriately called out as intentional.

Also, for this specific check, it's probably better, because it's more explicit, to say:

```ts
if (telegrafs.length > 0)
```
since we want to make sure that something comes back from the telegrafs api request, and technically if `telegrafs.length === -1`, it will pass `!= 0`.

It's perfectdly idiomatic JavaScript to just shorten it to

```ts
if (telegrafs.length)
```
however. Either one is acceptable.

Also, there's no need for an extra `else` since both these statements `return`. It's good to determine what the default/happy path is, and code around that.

```ts
if (telegrafs.length) {
  return telegrafPageLink
}

return newTelegrafPageLink
```

The difference here is subtle, but it's there. The `if/else` form gives equal weight to both choices, and doesn't provide a default or backup case. The `if some case, return early` form subtlely implies that the case the `if` statement is checking for is the exception, because the default case just returns. So think about which case is most likely what the default/happy case is, and make the other case the `if case -> return`